Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -403,54 +409,113 @@ public void JumpCheck()
 				//actor.GetAnimator().SetBool("isSliding", false);
 			}
 
-			ChangeState( ActorStates.Jumping );
+			// start falling
+			StartLateJumpTimer();
+
+			ChangeState( ActorStates.Falling );
 		}
+	}
 
-		if ( !IsInState( ActorStates.Grounded ) )
+	/**
+	 * Orients the model to face the direction of movement
+	 *
+	 * Orientation is reactive to velocity, meaning the
+	 * model will face the direction of movement, rather
+	 * than the input direction.
+	 */
+	void OrientSelf()
+	{
+		Quaternion desiredLook = transform.rotation;
+		if ( IsInState( ActorStates.Climbing ) )
 		{
-			jumpColCheckTimer += Time.deltaTime;
+			Quaternion.LookRotation( climbSurfaceNormal );
```

Comment:
Guessing this should be desiredLook = Quaternion.LookRotation( climbSurfaceNormal );